package.json

```diff
@@ -7,15 +7,17 @@
         "app/*",
         "client/**",
         "core/*",
-        "server/*"
+        "server/*",
+        "server/aws-lsp-codewhisperer/src.gen/@amzn/codewhisperer-streaming"
```

Comment:
Does streaming client have to be defined as separate workspace in the top-level of monorepo? If the purpose is to reuse it across several sub-packages, I think it should be better stored in a separate dedicated location, rathen than CodeWhisperer LSP server.

Reply:
Currently all different codewhisperer servers live under `server/aws-lsp-codewhisperer` directory so I don't see it as a problem for now. 

Also given this is a non-ideal way of creating the streaming client that would hopefully be replaced with a proper way, keeping the mess contained here doesn't look that bad to me

Reply:
Seems like nested workspace could help here (I mean define `codewhisperer-streaming` as a workspace inside of `server/aws-lsp-codewhisperer` instead of root of monorepo), but idk if that works properly. There're few but reports open about it, but perhaps we could verify? 
https://github.com/npm/cli/issues/4774